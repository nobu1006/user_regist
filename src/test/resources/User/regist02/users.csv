id,name,name_kana,email,password,zip_code,address,telephone,regist_user,regist_date,update_user,update_date,del_flg
