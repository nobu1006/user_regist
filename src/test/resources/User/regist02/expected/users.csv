name,name_kana,email,zip_code,address,telephone,del_flg
テスト太郎,テストタロウ,regist-test@example.com,111-1111,新宿区新宿,03-3333-3333,false